gymnasium/core.py

```diff
@@ -340,18 +338,18 @@ def __repr__(self):
         return str(self)
 
     @property
-    def unwrapped(self) -> Env:
+    def unwrapped(self) -> Env[ObsType, ActType]:
```

Comment:
Not necessarily true - consider the wrappers that discretize actions, or add pixel observations

Reply:
Good point, removed